browser/component_updater/brave_component_updater_configurator.cc

```diff
@@ -176,7 +176,7 @@ bool BraveConfigurator::EnabledBackgroundDownloader() const {
 }
 
 bool BraveConfigurator::EnabledCupSigning() const {
-  return false;
+  return configurator_impl_.EnabledCupSigning();
```

Comment:
Looks like its true by default: https://source.chromium.org/chromium/chromium/src/+/main:components/component_updater/configurator_impl.cc;l=136;drc=a150b50c0ff706af12c449c7fccd3cf2745e2061

Reply:
`BraveConfigurator` must implement `EnabledCupSigning` because it is declared in `Configurator`, which is abstract. I don't think we can remove `BraveConfigurator::EnabledCupSigning()`.

Reply:
Oh, I thought this already subclassed something that overrode it.

Reply:
seems the problem is that we're not subclassing `ChromeConfigurator`, but we should be. Most of the code appears to be duplicated. I guess it's up to you if you want to fix that here or we can do it in a separate PR, but `BraveConfigurator` should sublcass `ChromeConfigurator`

Reply:
I'll do it separately, in https://github.com/brave/brave-browser/issues/41170.